### PR TITLE
Do materials for Feb resume workshop

### PR DESCRIPTION
**Pull request by KristinaRiemer**
# Title: 

## Pre-workshop checklist:

-   [ ] Create a Zoom meeting with registration
-   [ ] Write up announcement and record on the [Workshops HackMD](https://hackmd.io/-hSaZbuzQI-gAPP3pASvRw)
-   [ ] Advertise
    -   [ ] Edit upcoming workshops section of [group website workshops page](https://datascience.cals.arizona.edu/workshops)
    -   [ ] Add to Data Science Institute calendar <https://datascience.arizona.edu/calendar>
    -   [ ] Send out announcement to mailing list
    -   [ ] Twitter
-   [ ] Provide homework / background reading as appropriate (installation, references, etc.)
-   [ ] Send out reminders
-   [ ] Create pre- and post-workshop surveys
-   [ ] Send out pre-workshop survey [WIP]

Full checklist here: https://osf.io/tzmhp/wiki/Workshops/

## Materials

Create a folder in the format YYYYMM-short-title and include:

- [ ] Slides (if applicable)
- [ ] Notes

Use .md, .Rmd with `output: github_document`, or .qmd with `format: gfm` OR render to RStudio Connect or something like that.

### CODE REVIEW

File: 202402-cv-resume-r/2024-02-cv-resume-r.qmd [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+If you can't install `datadrivencv`, we have a [template repo](https://github.com/cct-datascience/template_resume_datadrivencv?tab=readme-ov-file) of needed files
+
+# Let's check everyone's installations!
+
+## Generate initial files
+
+```         
+library(datadrivencv)
+use_datadriven_cv(full_name = "Your Name", data_location = "https://docs.google.com/spreadsheets/d/1mq7H8ZiF6k7H_f6rY2eA0sq2Xyxf9msmXv3LPsvSYoo/edit#gid=1730172225")
+```
```
Comment (Aariq) [inline 202402-cv-resume-r/2024-02-cv-resume-r.qmd:124]: Use ```r to format code like R code
```suggestion
``` r
library(datadrivencv)
use_datadriven_cv(full_name = "Your Name", data_location = "https://docs.google.com/spreadsheets/d/1mq7H8ZiF6k7H_f6rY2eA0sq2Xyxf9msmXv3LPsvSYoo/edit#gid=1730172225")
```
```

**Review by Aariq — COMMENTED** [review]
Just some comments in case you're interested

File: 202402-cv-resume-r/2024-02-cv-resume-r.qmd [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+-   Image
+
+-   Icons
+
+Multiple places where they could be changed: `cv.rmd`, `dd_cv.css`, data files
+
+Takes some playing around & trial and error
+
+# What do you want to see changed?
```
[the remark this answers is not in the document]
Comment (KristinaRiemer) [reply 202402-cv-resume-r/2024-02-cv-resume-r.qmd:199]: I would have to learn how to do this myself! Would be worth at least knowing if someone asks a question that would require this.

File: 202402-cv-resume-r/2024-02-cv-resume-r.qmd [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+::: incremental
+We *will* be learning:
+
+-   The pros and cons of different R tools for creating resumes
+-   How to create and customize resumes the `datadrivencv` R package
+
+::: callout-note
+You will leave this workshop with a partially complete, custom resume!
+:::
+:::
```
[the remark this answers is not in the document]
Comment (KristinaRiemer) [reply 202402-cv-resume-r/2024-02-cv-resume-r.qmd]: Yes! Thanks!

File: 202402-cv-resume-r/2024-02-cv-resume-r.qmd [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+
+## Why use R to do this?
```
[the remark this answers is not in the document]
Comment (KristinaRiemer) [reply 202402-cv-resume-r/2024-02-cv-resume-r.qmd:76]: Added this in as a new slide a couple slides later.

**Pull request merged**